### ISSUE TITLE
Evaluation not working on distilbert-base-uncased-distilled-squad

### ISSUE THREAD

**Issue by graviraja** [huggingface/transformers#2449]
## 🐛 Bug

Model I am using DistilBert: distilbert-base-uncased-distilled-squad

Language I am using the model on English:

The problem arise when using:
* [x] the official example scripts: run_squad.py in examples

The tasks I am working on is:
* [x] an official GLUE/SQUaD task: SQuAD 1.1 and SQuAD2.0 dev dataset

## To Reproduce

1. python run_squad.py --model_type distilbert     --model_name_or_path distilbert-base-uncased-distilled-squad     --do_eval     --do_lower_case     --predict_file $SQUAD_DIR/dev-v2.0.json --max_seq_length 384     --doc_stride 128     --output_dir ./distill_squad/     --per_gpu_eval_batch_size=4    --version_2_with_negative

2. python run_squad.py --model_type distilbert     --model_name_or_path distilbert-base-uncased-distilled-squad     --do_eval     --do_lower_case     --predict_file $SQUAD_DIR/dev-v1.1.json --max_seq_length 384     --doc_stride 128     --output_dir ./distill_squad/     --per_gpu_eval_batch_size=4  

## Expected behavior
Results on evaluation data
```python
{
"exact": 80.4177545691906,
"f1": 84.07154997729623,
"total": 11873,
"HasAns_exact": 76.73751686909581,
"HasAns_f1": 84.05558584352873,
"HasAns_total": 5928,
"NoAns_exact": 84.0874684608915,
"NoAns_f1": 84.0874684608915,
"NoAns_total": 5945
}
```

## Environment

* OS: CentOS Linux
* Python version: 3.6.9
* PyTorch version: 1.3.1
* PyTorch Transformers version (or branch): 2.3.0
* Using GPU: yes
* Distributed or parallel setup: None
* Any other relevant information:

## Additional context
```code
01/08/2020 08:41:52 - INFO - __main__ -   ***** Running evaluation  *****
01/08/2020 08:41:52 - INFO - __main__ -     Num examples = 10833
01/08/2020 08:41:52 - INFO - __main__ -     Batch size = 4
Evaluating:   0%|                                                                                                                              | 0/2709 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "run_squad.py", line 815, in <module>
    main()
  File "run_squad.py", line 804, in main
    result = evaluate(args, model, tokenizer, prefix=global_step)
  File "run_squad.py", line 323, in evaluate
    outputs = model(**inputs)
  File "/media/data2/anaconda/envs/distill2/lib/python3.6/site-packages/torch/nn/modules/module.py", line 541, in __call__
    result = self.forward(*input, **kwargs)
TypeError: forward() got an unexpected keyword argument 'token_type_ids'
```

I just changed the model to bert and model_name to bert-base-uncased. It is working fine. I think there is some problem with distilbert model. Can you please help me on this?

**Comment by LysandreJik** [comment]
Hello, thanks for raising this issue! This should have been fixed with 16ce15e, can you let me know if it fixes your issue?

**Comment by graviraja** [comment]
Hi @LysandreJik, thanks for fixing the issue on such short notice.  Yes, now it's working.
